devel/swig3/Portfile

```diff
@@ -0,0 +1,335 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem 1.0
+
+name            swig3
+version         3.0.12
+revision        0
+checksums       rmd160 41877e9de3ff598731ef36161f77fa66dec3c301 \
+                sha256 7cf9f447ae7ed1c51722efc45e7f14418d15d7a1e143ac9f09a668999f4fc94d
+
+categories      devel
+maintainers     {michaelld @michaelld} openmaintainer
+
+# The swig tool itself is GPL, but code incorporated into dependents has a
+# permissive license.
+license         {GPL-3+ Permissive}
+
+description     Tool to wrap C/C++ libraries in high-level languages.
+long_description \
+    SWIG is a software development tool that connects programs written in C \
+    and C++ with a variety of high-level programming languages. SWIG is \
+    primarily used with common scripting languages such as Perl, Python, \
+    Tcl/Tk, Ruby, Guile and MzScheme, however the list of supported languages \
+    also includes non-scripting languages such as Java and OCAML. SWIG is \
+    most commonly used to create high-level interpreted programming \
+    environments, user interfaces, and as a tool for testing and \
+    prototyping C/C++ software. SWIG may be freely used, distributed, and \
+    modified for commercial and noncommercial use.
+
+homepage        http://www.swig.org/
+platforms       darwin
+master_sites    sourceforge:project/swig/swig/swig-${version}
+
+distfiles       swig-${version}${extract.suffix}
```

Comment:
Can you add in ```dist_subdir     swig``` here? That way there is continuity to where SWIG source code tarballs are stored?